app/survey_config/business_config.py

```diff
@@ -20,6 +20,15 @@ def __post_init__(self):
         self.base_url = self._stripped_base_url
         super().__post_init__()
 
+        if self.schema:
+            self.data_layer: list[dict] = [
+                {
+                    key: self.schema.json[key]
+                    for key in ["form_type", "survey_id", "title"]
+                    if key in self.schema.json
+                }
+            ]
```

Comment:
Could use walrus operator:
```suggestion
            self.data_layer: list[dict] = [
                {
                    key: value
                    for key in ["form_type", "survey_id", "title"]
                    if (value := self.schema.json.get(key))
                }
            ]
```